src/__tests__/feature/flow_chart_panel/views/__snapshots__/ControlBar.test.tsx.snap

```diff
@@ -31,6 +32,35 @@ exports[`Controls should render correctly: __main__ 1`] = `
+        >
+          <div>
+            <svg
+              fill="none"
+              height="17"
+              viewBox="0 0 11 11"
+              width="17"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M4.01032 10.9167L3.79365 9.18334C3.67629 9.1382 3.5657 9.08404 3.46188 9.02084C3.35806 8.95765 3.2565 8.88994 3.15719 8.81772L1.54574 9.4948L0.0561523 6.92188L1.45094 5.86563C1.44192 5.80244 1.4374 5.7415 1.4374 5.68282V5.3172C1.4374 5.25852 1.44192 5.19758 1.45094 5.13438L0.0561523 4.07813L1.54574 1.50522L3.15719 2.1823C3.2565 2.11008 3.36032 2.04237 3.46865 1.97918C3.57699 1.91598 3.68532 1.86182 3.79365 1.81668L4.01032 0.0833435H6.98949L7.20615 1.81668C7.32351 1.86182 7.4341 1.91598 7.53792 1.97918C7.64174 2.04237 7.74331 2.11008 7.84261 2.1823L9.45407 1.50522L10.9437 4.07813L9.54886 5.13438C9.55789 5.19758 9.5624 5.25852 9.5624 5.3172V5.68282C9.5624 5.7415 9.55338 5.80244 9.53532 5.86563L10.9301 6.92188L9.44053 9.4948L7.84261 8.81772C7.74331 8.88994 7.63949 8.95765 7.53115 9.02084C7.42282 9.08404 7.31449 9.1382 7.20615 9.18334L6.98949 10.9167H4.01032ZM5.52699 7.39584C6.0506 7.39584 6.49747 7.21077 6.86761 6.84063C7.23775 6.4705 7.42282 6.02362 7.42282 5.50001C7.42282 4.9764 7.23775 4.52952 6.86761 4.15938C6.49747 3.78925 6.0506 3.60418 5.52699 3.60418C4.99435 3.60418 4.54522 3.78925 4.17959 4.15938C3.81397 4.52952 3.63115 4.9764 3.63115 5.50001C3.63115 6.02362 3.81397 6.4705 4.17959 6.84063C4.54522 7.21077 4.99435 7.39584 5.52699 7.39584ZM5.52699 6.31251C5.30129 6.31251 5.10945 6.23352 4.95147 6.07553C4.79348 5.91754 4.71449 5.7257 4.71449 5.50001C4.71449 5.27432 4.79348 5.08248 4.95147 4.92449C5.10945 4.7665 5.30129 4.68751 5.52699 4.68751C5.75268 4.68751 5.94452 4.7665 6.10251 4.92449C6.26049 5.08248 6.33949 5.27432 6.33949 5.50001C6.33949 5.7257 6.26049 5.91754 6.10251 6.07553C5.94452 6.23352 5.75268 6.31251 5.52699 6.31251ZM4.95824 9.83334H6.02803L6.21761 8.39793C6.49747 8.3257 6.75702 8.21963 6.99626 8.0797C7.23549 7.93977 7.45442 7.7705 7.65303 7.57188L8.99365 8.12709L9.52178 7.20626L8.35719 6.32605C8.40233 6.19966 8.43393 6.0665 8.45199 5.92657C8.47004 5.78664 8.47907 5.64445 8.47907 5.50001C8.47907 5.35557 8.47004 5.21338 8.45199 5.07345C8.43393 4.93352 8.40233 4.80036 8.35719 4.67397L9.52178 3.79376L8.99365 2.87293L7.65303 3.44168C7.45442 3.23404 7.23549 3.06025 6.99626 2.92032C6.75702 2.78039 6.49747 2.67432 6.21761 2.60209L6.04157 1.16668H4.97178L4.78219 2.60209C4.50233 2.67432 4.24278 2.78039 4.00355 2.92032C3.76431 3.06025 3.54539 3.22952 3.34678 3.42813L2.00615 2.87293L1.47803 3.79376L2.64261 4.66043C2.59747 4.79584 2.56587 4.93126 2.54782 5.06668C2.52976 5.20209 2.52074 5.34654 2.52074 5.50001C2.52074 5.64445 2.52976 5.78439 2.54782 5.9198C2.56587 6.05522 2.59747 6.19064 2.64261 6.32605L1.47803 7.20626L2.00615 8.12709L3.34678 7.55834C3.54539 7.76598 3.76431 7.93977 4.00355 8.0797C4.24278 8.21963 4.50233 8.3257 4.78219 8.39793L4.95824 9.83334Z"
```

Comment:
Instead of making this inline, you can create a component or SVG file and import it.